ai-platform-unified/notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb

```diff
@@ -1,1092 +1,1335 @@
 {
```

Comment:
<p>This looks like the same introduced cut-and-paste (?) error as in the control flow notebook-- this image doesn't belong to this notebook.</p><p> </p><p>That said, this notebook probably SHOULD include a screenshot of the pipeline run, particularly b/c in this case it produces some nice metrics viz.  Apparently I neglected to do that with my original version of the notebook. Perhaps when you're updating you could grab a screenshot and add it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>